src/types/types.ts

```diff
@@ -295,6 +299,15 @@ export type TaskObject = Omit<EnqueuedTaskObject, 'taskUid'> & {
 
     // Distinct attribute on settings actions
     distinctAttribute: DistinctAttribute
+
+    // Number of tasks that matched the originalQuery filter
+    matchedTasks?: number
+
+    // Number of tasks that were canceled
+    canceledTasks?: number
+
+    // Query parameters used to filter the tasks
+    originalQuery?: string
```

Comment:
Just a reminder, that this will change in the next RC.

Reply:
yes! It's written in the body